cases-SEP24/sep10.test.js

```diff
@@ -49,7 +49,13 @@ const getAccount = (function() {
 beforeAll(async () => {
   if (process.env.MAINNET === "true" || process.env.MAINNET === "1") {
     let kps = [];
-    for (let i = 0; i < 10; i++) kps.push(StellarSDK.Keypair.random());
+    for (let i = 0; i < 10; i++) {
+      let kp = StellarSDK.Keypair.random();
+      process.stdout.write(
```

Comment:
Whats the difference between `process.stdout.write()` vs `console.log()`? Can we make sure we're only using one or the other if there is no functional difference?

Reply:
The functional difference is that it doesn't write to stdout the following
```
console.log cases-SEP6/sep10.test.js:69
    Generated keypair GC6EJLOZXV6M5KJFFVDER2TYCOTYAZI76IXLRWXJXGHGQUYI6HGW3VQP:SCQBJRBNVPZXTQUA7N7BY4GPU5IGLD4MPQZKPZNKGIJONADSED5QDQE5
```
where instead it writes to stdout the following
```
Generated keypair GC6EJLOZXV6M5KJFFVDER2TYCOTYAZI76IXLRWXJXGHGQUYI6HGW3VQP:SCQBJRBNVPZXTQUA7N7BY4GPU5IGLD4MPQZKPZNKGIJONADSED5QDQE5
```
Which is a lot cleaner when we print several keypairs to stdout.

Reply:
I just found a bug with my method.
When running the app's client it doesnt play nice directly writing to stdout.
```
[1] undefined:2
[1] Generated keypair GCP6HR5NNFONYJ6BSSP2W5UIN26CH25QJFKHTNDGTRR6TV7UMKIWIV2G:SCLHI72OQMTUCXF44YCG5C4BF5DDVF7KNIFEQR3K2VVK57XVXLQ6T5M6
[1] ^
[1]
[1] SyntaxError: Unexpected token G in JSON at position 2
[1]     at JSON.parse (<anonymous>)
[1]     at ChildProcess.<anonymous> (/Users/lawrence/stellar-dev/transfer-server-validator/server/jest-controller/run-test.js:30:28)
[1]     at ChildProcess.emit (events.js:315:20)
[1]     at maybeClose (internal/child_process.js:1048:16)
[1]     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
[1] [nodemon] app crashed - waiting for file changes before starting...
```
I have a change coming to accommodate this by just using console.log instead despite my distain for the noise it generated.
Using `console.log` in lieu of `process.stdout.write` looked like this
```
[1]   console.log cases-SEP6/sep10.test.js:69
[1]     Generated keypair GBY5YABOKYBYSX7IA5YAOD3GJC4YLXEUXFX2UXRGY3UQPEFXI4LYFIO7:SBIOR4YLJIQRWIEIDPYT6OEHZCCBXS6E2L2X6XSCFELJQHRZYPRW272C
[1]
[1]
[1]
[1]   console.log cases-SEP6/sep10.test.js:69
[1]     Generated keypair GA4MWBXQGFL25DKA67AIWQMEB5TIBUR2JS3QE4U52N2ALWSUMALTD2IY:SD4WAGFKSQBA4C5EPUSDDS5GQWQ7LVSTRQW2X76ZKUBQV55XLHAND7VC
[1]
[1]
[1]
[1]   console.log cases-SEP6/sep10.test.js:69
[1]     Generated keypair GCXJPIREDU3ZW6YOKBYLEERTPLLBLKUK5KAEKMVCYMVRWEYPWIGBQCVB:SDAI4JQRCVJR3QUN7PNYZSGKZJL34MMVJGBEBQALBMJKQPG7QTFAGAT4
...
``` 
But I wrote a solution to appear like I originally intended 
So now it looks like
```
[1]   console.log cases-SEP6/sep10.test.js:71
[1]     Generated keypair GBXDFFYIVIKUVOZ6U5QITPW4A4SQCIWIZBTYVQBJFNHIFL3HIFNLGKHW:SASBDA4IU3GQVMWWQT5WAC4LPO2RUW3SV4YRRNIJT4E2WZ7CHSGBIODK
[1]     Generated keypair GCN7YORUY5HLGAKXMWFXVYDA6PEPHT2QXNTHA5DDEWFTUSUGIZQKDT6N:SBUTQCFRKRPKT2VKMR5DWJH4CQGPNM7F2I3DQNPFXCOQBPRRLX3OY45E
[1]     Generated keypair GB7JZ36TN3D7MHZHNFW64B7BIGUTRTEBV77KDJ3O2ZIWMYBPBSQGESIB:SAAKEIAYQRPIKZBVCYSGB7T2RKCP6ILCLEAWXCRSQQF4RD3IB3D5NPTQ
...
```
And no bugs!(I hope!)